packages/features/shell/Shell.tsx

```diff
@@ -785,39 +780,92 @@ type SideBarContainerProps = {
 
 type SideBarProps = {
   bannersHeight: number;
+  user?: UserAuth | null;
 };
 
 function SideBarContainer({ bannersHeight }: SideBarContainerProps) {
-  const { status } = useSession();
+  const { status, data } = useSession();
   const router = useRouter();
 
   // Make sure that Sidebar is rendered optimistically so that a refresh of pages when logged in have SideBar from the beginning.
   // This improves the experience of refresh on app store pages(when logged in) which are SSG.
   // Though when logged out, app store pages would temporarily show SideBar until session status is confirmed.
   if (status !== "loading" && status !== "authenticated") return null;
   if (router.route.startsWith("/v2/settings/")) return null;
-  return <SideBar bannersHeight={bannersHeight} />;
+  return <SideBar bannersHeight={bannersHeight} user={data?.user} />;
```

Comment:
If there isn't a user session we shouldn't render the side bar. Example the App store is available publicly but with no user logged in the sidebar still renders.

@sean-brydon side not the verify email banner still shows up on top.

<img width="1314" alt="image" src="https://github.com/calcom/cal.com/assets/65426560/1c22966c-a63d-4c87-a8bb-dac1ac34517a">

Reply:
I can't see the sidebar when I'm not authenticated, in the line 793 return null if the session status !== "authenticated" `  if (status !== "loading" && status !== "authenticated") return null;`

Reply:
I don't see the issue.